src/package.json

```diff
@@ -76,7 +75,7 @@
     "@angular/compiler": ">=12.0.0 <14.0.0",
     "@angular/core": ">=12.0.0 <14.0.0",
     "@angular/forms": ">=12.0.0 <14.0.0",
-    "chart.js": ">2.9.0 <3.0.0",
-    "ng2-charts": ">=2.3.0 <3.0.0"
+    "chart.js": ">2.0.0 <4.0.0",
+    "ng2-charts": ">2.0.0 <4.0.0"
```

Comment:
Was something changed to support older versions of these dependencies as well? If not, I would suggest maintaining the previous lower bounds of these ranges.